File: src/test/java/org/cloudfoundry/credhub/util/DatabaseUtilities.java
package org.cloudfoundry.credhub.util;

public final class DatabaseUtilities {
  private DatabaseUtilities() {
    super();
  }

  public static byte[] getExceedsMaxBlobStoreSizeBytes() {
    final int exceedsMaxBlobStoreSize = 70000;
    final byte[] exceedsMaxBlobStoreValue = new byte[exceedsMaxBlobStoreSize];
    for (int i = 0; i < exceedsMaxBlobStoreSize; i++) {
      final byte randomNumber = (byte) Math.round(Math.random() * 10);
      exceedsMaxBlobStoreValue[i] = randomNumber;
    }

    return exceedsMaxBlobStoreValue;
  }
}

File: components/test-support/src/test/kotlin/org/cloudfoundry/credhub/utils/DatabaseUtilities.kt
package org.cloudfoundry.credhub.utils

import kotlin.math.roundToInt

class DatabaseUtilities {
    private constructor()

    companion object {
        fun getExceedsMaxBlobStoreSizeBytes(): ByteArray {
            val exceedsMaxBlobStoreSize = 70000
            val exceedsMaxBlobStoreValue = ByteArray(exceedsMaxBlobStoreSize)
            for (i in 0 until exceedsMaxBlobStoreSize) {
                val randomNumber = (Math.random() * 10).roundToInt().toByte()
                exceedsMaxBlobStoreValue[i] = randomNumber
            }
            return exceedsMaxBlobStoreValue
        }
    }
}